script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -787,7 +787,7 @@ Panting as I fight to control my breathing, I grab hold of her arm.
 [sha:b5ee40084a155aa50fe025f5f02a75cba9783634]{
 -- Page 326, Offset 17382.
 -- 「……なによ。そういう志貴だって、いまにも倒れそうな顔してる」
-"...What the hell.
+"...Huh.
```

Comment:
"What the hell" sounds overexaggerated.

Reply:
Eh, I still think revert. `Huh` is just so gormless.